biojava-structure/src/main/java/org/biojava/nbio/structure/basepairs/MismatchedBasePairParameters.java

```diff
@@ -0,0 +1,85 @@
+package org.biojava.nbio.structure.basepairs;
+
+import org.biojava.nbio.structure.Atom;
+import org.biojava.nbio.structure.Chain;
+import org.biojava.nbio.structure.Group;
+import org.biojava.nbio.structure.Structure;
+
+import javax.vecmath.Matrix4d;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Contributed to BioJava under its LGPL
+ * This class allows for finding inter-strand base pairs that are not necessarily canonical Watson-Crick pairs.
+ * The implementation of findPair is different than that of the base class.
+ * Created by luke on 7/23/17.
+ */
```

Comment:
Minor comment: could you use @author annotations (here an in other new classes that you introduced).